EliteDangerous/EDSM/EDSMClass.cs

```diff
@@ -270,8 +270,16 @@ internal long GetNewSystems(Func<bool> cancelRequested, Action<int, string> repo
                     break;
                 }
 
-                updates += SystemClassEDSM.ParseEDSMUpdateSystemsString(json, ref lstsyst, ref outoforder, false, cancelRequested, reportProgress, false);
-                lstsystdate += TimeSpan.FromHours(12);
+                long cnt = SystemClassEDSM.ParseEDSMUpdateSystemsString(json, ref lstsyst, ref outoforder, false, cancelRequested, reportProgress, false);
+                updates += cnt;
+                if (cnt < 100)
```

Comment:
Was that supposed to be 1000?

Reply:
I originally had it as 1000, but I decided to reduce it in case the `LIMIT 1000` includes some rows that are filtered out.